modules/services/espanso.nix

```diff
@@ -0,0 +1,78 @@
+{ pkgs, config, lib, ... }:
+let
+  inherit (lib)
+    mkOption mkEnableOption mkIf maintainers literalExpression types platforms;
+    inherit (lib.hm.assertions) assertPlatform;
+  cfg = config.services.espanso;
+  yaml = pkgs.formats.yaml { };
+in {
+  meta.maintainers = with maintainers; [ lucasew ];
+
+  options = {
+    services.espanso = {
+      enable = mkEnableOption "Espanso: cross platform text expander in Rust";
+      package = mkOption {
+        type = types.package;
+        description = "Which espanso package to use";
+        default = pkgs.espanso;
+        defaultText = literalExpression "pkgs.espanso";
+      };
+      config = mkOption {
+        type = yaml.type;
+        description = "Espanso configuration";
+        default = { matches = [ ]; };
+        example = literalExpression ''
```

Comment:
Would it make sense to provide a fallback mechanism to allow using a file or a multi-line string directly?